docs/how-to-guides/jdbc-cosmos-notes.md

```diff
@@ -62,6 +62,20 @@ client.get_offline_features(...)
 
 These values will be automatically passed to the Feathr core when submitting the job.
 
+If you want to use token, the code will be like this:
+```python
+src_name="source_name"
```

Comment:
Where should user get this `auth="TOKEN"` part?

Reply:
will add a snippet of `how to generate token` and add a skipped test case for now.